_posts/2018-06-03-resultant-theory.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title:  "Resultants in SymPy"
+date:   2018-06-03
+comments: False
+---
+
+I have recently been doing some work related to systems of polynomials. This work
+has lead me to study several academics papers on **resultant theory**. In this blog
+post I aiming to give an introduction to resultant theory and a demonstration of
+how we can use the Python package [SymPy](http://www.sympy.org/en/index.html)
+to calculate the resultant of systems.
```

Comment:
`I will end the blog post by describing the particular contribution I've recently made to [Sympy](), Python library for symbolic mathematics.`

(Or something like that)